schema/release-schema.json

```
@@ -33,7 +33,7 @@
     },
     "tag": {
       "title": "Release Tag",
-      "description": "A tag labeling the release, using the the open [releaseTag](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#release-tag) codelist. Tags distinguish, for example, planning and contracting processes and the stages of contracting processes. Codes outside the releaseTag codelist might indicate, for example, the notice or form to which the release corresponds, or the event that triggered the publication of the release. Planning processes must not use the 'tender' code, even if they use `tender` fields.",
+      "description": "A tag labeling the release, using the the open [releaseTag](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#release-tag) codelist. Tags distinguish, for example, planning and contracting processes and the stages of contracting processes. Codes outside the releaseTag codelist may indicate, for example, the notice or form to which the release corresponds, or the event that triggered the publication of the release. A planning process must use either the 'planning' or 'planningUpdate' code, and must not use any other code from the releaseTag codelist. A contracting process should not use the 'planning' or 'planningUpdate' codes.",
```

Fix the "the the" repetition in the tag description.

Did we discuss changing 'might' to 'may' in the sentence about codes from outside the codelist?

I think using a normative keyword is inappropriate because the sentence doesn't set a normative rule about the permitted semantics of codes from outside the codelist, rather it provides some illustrative examples.

```suggestion
      "description": "A tag labeling the release, using the open [releaseTag](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#release-tag) codelist. Tags distinguish, for example, planning and contracting processes and the stages of contracting processes. Codes outside the releaseTag codelist might indicate, for example, the notice or form to which the release corresponds, or the event that triggered the publication of the release. A planning process must use either the 'planning' or 'planningUpdate' code, and must not use any other code from the releaseTag codelist. A contracting process should not use the 'planning' or 'planningUpdate' codes.",
```

---

I looked and looked and couldn't see the "the the" that was being referred to! thanks for finding it :)

---

we hadn't discussed it, I was just going by the guidance that everything in the docs/schema folder is normative therefore we should use normative language. I'll take your suggestion, James can change it back if he doesn't agree